clients/sample-dialect-vscode/.vscode/settings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "cobol-lsp.smart-tab": {
```

Comment:
Does it make sense to include any of this?

Reply:
Removed